debian/cloud-init.maintscript

```diff
@@ -7,3 +7,4 @@ rm_conffile /etc/init/cloud-init-nonet.conf 0.7.9-243-ge74d775-0ubuntu2~
 rm_conffile /etc/init/cloud-init.conf 0.7.9-243-ge74d775-0ubuntu2~
 rm_conffile /etc/init/cloud-log-shutdown.conf 0.7.9-243-ge74d775-0ubuntu2~
 rm_conffile /etc/NetworkManager/dispatcher.d/hook-network-manager 23.3~4ga31370af-0ubuntu1~
+rm_conffile /etc/cloud/clean.d/README 24.1~3gb729a4c4-0ubuntu1~
```

Comment:
Turns out we actually need this version to be matching the next expected release version. Since the devel release will adhere to either 24.1~4 if we publish another snapshot prior to upstream release of 24.1, or it'll be `24.1` our best best to match both cases is below

 ```suggestion
rm_conffile /etc/cloud/clean.d/README 24.1~4~
```

Reply:
Thanks for this. I will update the maintscripts.